File: customlistviewsample/src/main/java/tech/thdev/customlistviewsample/data/GitHubItem.java
package tech.thdev.customlistviewsample.data;

/**
 * Created by tae-hwan on 10/24/16.
 * <p>
 * GitHub api <a href="https://api.github.com/search/users?q=tom+repos:%3E42+followers:%3E1000">GitHub User search sample</a>
 */

public class GitHubItem {
    public String login;
    public String avatar_url;
    public String html_url;
    public float score;
}

File: app/src/main/java/tech/thdev/app/data/GitHubItem.kt
package tech.thdev.app.data

import com.google.gson.annotations.SerializedName

/**
 * Created by tae-hwan on 10/24/16.
 *
 * GitHub api [GitHub User search sample](https://api.github.com/search/users?q=tom+repos:%3E42+followers:%3E1000)
 */
class GitHubItem {
    @SerializedName("login")
    val login: String? = null
    @SerializedName("avatar_url")
    val avatarUrl: String? = null
    @SerializedName("html_url")
    val htmlUrl: String? = null
    @SerializedName("score")
    val score: Float = 0f
}